how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,248 @@
+| Podman          | WIP     | [see here](https://github.com/containers/libpod/blob/master/install.md)
+| `slirp4netns`   | 0.4.0   | [see here](https://github.com/rootless-containers/slirp4netns#quick-start)
+| Kata Containers | WIP     | [see here](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+> **NOTE:**
+>
+> If installing Podman with a package manager, there is usually no need to
+> install `slirp4netns` separately.
+
+## Configuration
+
+Now that you have installed Kata Containers and Podman, you need to configure
+them for rootless execution.
+
+### Disable SELinux
+
+> **Warning:**
+>
+> You should understand the security implications before disabling SELinux.
+
+If SELinux is installed and enabled, it needs to be disabled with the
+following command (Kata Containers
+[does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)).
+
+> **Warning:**
+> The following command might differ depending on the distro you use:
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+### Add user to KVM group
+
+If running a KVM based hypervisor, add the user running the workload to the KVM group:
+
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+
+### Reboot
+
+Reboot the system for the changes to take effect (when you disable SELinux you
+must reboot, while logging out and back in is enough to have that user joining
+the `kvm` group).
+
+Verify the configuration is correct:
+
+- If installed, disable SELinux:
+  ```bash
+  $ getenforce
+  Disabled
+  ```
+
+- The user should be in the `kvm` group:
+
+  ```
+  $ groups | grep -ow kvm
+  kvm
+  ```
+
+### Setup Kata configuration files
+
+If the Kata `configuration.toml` file does not exist in `/etc`, create it:
+
+```bash
+$ sudo mkdir -p /etc/kata-containers/
+$ sudo install -D -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers/
+```
+
+Or, if the file exists, but is not readable by the user:
+
+```bash
+$ sudo chown -R a+r /etc/kata-containers/
+```
+
+### Disable `vhost-net`
+
+Disable `vhost-net` in the Kata configuration file, by commenting out the
+`disable_vhost_net` line:
+
+```bash
+$ sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' /etc/kata-containers/configuration.toml
+```
+
+### Modify the Kata images permissions
+
+The Kata images needs to be accessible by the user, so change the permissions
+of the image directory to be readable by the user.
+
+```bash
+$ sudo chown -R a+r /usr/share/kata-containers
+```
+
+### Set up Podman rootless configuration
+
+If `libpod.conf` does not exist in `~/.config/containers/`:
+
+```bash
+$ [ -e ~/.config/containers/libpod.conf ] || install -D /usr/share/containers/libpod.conf ~/.config/containers/libpod.conf
+```
+
+By default the `tmp_dir` in `libpod.conf` is set to `/var/run/libpod`, however
```

Comment:
Add a comma after "default"
By default, the ....

Reply:
On my Fedora system this was already set to `/run/user/1000`  I'd just reword a bit   "Check the `tmp_dir` in `libpod.conf` is set to `/run/user/$(id -u)`.  If not then change it with the following command.